dart-impl/mpi/include/dash/dart/mpi/dart_globmem_priv.h

```diff
@@ -6,6 +6,7 @@
 #define DART_GPTR_COPY(gptr_, gptrt_)                       \
   do {                                                      \
     gptr_.unitid = gptrt_.unitid;                           \
+    gptr_.flags  = gptrt_.flags;                            \
```

Comment:
I can't make up a reason for this macro, either. It would only make sense if `dart_gptr_t` contained dynamically allocated fields, and that again would be ... preposterous.